src/Server/Event/DoctrineResourceEvent.php

```diff
@@ -51,19 +51,17 @@ class DoctrineResourceEvent extends Event
     /**
      * @return ObjectManager
      */
-    public function getObjectManager()
+    public function getObjectManager(): ObjectManager
     {
         return $this->objectManager;
     }
 
     /**
      * @return $this
```

Comment:
This is in conflict with the actual return type? :thinking: 

Still, API change is stark, and requires a major version